README.md

```diff
@@ -23,6 +23,17 @@ It was created as an answer to deeply recursive Pyanodon recipes, which the tool
 
 Among other things YAFC has Never Enough Items, which is FNEI on steroids. In addition to showing the recipes, it shows which ones you probably want to use, and how much.
 
+## Getting Started
+
+YAFC is a desktop app. Windows build is the most tested, but OSX and Linux are there too. See [Linux and OSX installation instructions](/Docs/LinuxOsxInstall.md).
+
+1. Navigate to the [Releases](https://github.com/have-fun-was-taken/yafc-ce/releases),
+2. Download the zip file for your given OS,
+3. Extract the zip file to your preferred location,
+4. If on OSX/Linux, make sure to make `YAFC` executable via `chmod +x ./YAFC`
```

Comment:
```suggestion
```
Linux instructions got updated, so this line can be removed. Essentially, every Linux/OSX user is expected to follow the link above for Linux/OSX installation docs.

Reply:
I think it will be better to move this to the Linux installation instructions, instead of fully removing. I'll see if I can make a PR for this without having the repo forked yet (my reuquest to move the 'base repo' is still pending)